molgenis_documentation/user_documentation.md

```diff
@@ -947,6 +951,163 @@ To get the feel of how certain styles look, you can select it in the dropdown me
+The upload module is the place in MOLGENIS where you can upload your data into the MOLGENIS application.
+
+#### <a name="upload-possibilities"></a>Upload possibilities
+The MOLGENIS upload module supports the following upload files formats and data:
+	
+|file format		|file extention             |data formats|
+||
+|CSV              	|".csv" ".txt" ".tsv" ".zip"|EMX|
+|Excel            	|".xls" ".xlsx"             |EMX|
+|OWL              	|".owl.zip"                 |OWL|
+|VCF (version 4.0)	|"vcf" ".vcf.gz"            |VVCF (version 4.0)|:
+
+##### <a name="upload-possibilities-abbreviations"></a>Abbreviations:
+
+* CSV: Comma Seperated Value
+* OWL: Web Ontology Language
+* VCF: Variant Call Format
+* EMX: Entity Model Extensible
+
+#### <a name="upload-screens"></a>Upload pages
+The different pages will be explianed by uploading the<a name="advanced-data-example"></a> "Advanced data example" ([download](download/advanced_data_example_v20151104.xlsx)) example data set.
+
+The pages are: 
+
+1. Upload file
+2. Options
+3. Packages
+4. Validation
+5. Result
+
+Navigation buttons at the bottom of the pages:
+
+* Previous: Go to the previous page.
+* Next: Go to the next page.
+* Restart: Push this button when you want to start importing a new data set. It will redirect you to the start of this wizard. Pushing this button will kill the session but not the importing job.
+* Finish: The same as Restart.
+
+
+##### <a name="upload-screens-upload-file"></a>Upload file page
+
+###### <a name="upload-screens-options-steps"></a>Steps
+
+1. Select a file to upload.
+2. Click on the next button.
+
+![Upload file screen](images/plugin-upload/upload-file-screen.png?raw=true, "Upload file")
+
+##### <a name="upload-screens-options"></a>Options page
+Select a data upload option. On this page you can select the rules of how to upload your data into MOLGENIS. Because this dataset is an unknown dataset to the application we leave the default option "Add entities" selected. In tabular data sets, the term entities refers to data-rows.
+It is important to understand that this selection is about the data and not the meta data of the data set.
+
+###### <a name="upload-screens-options-options"></a>Options
+
+1. Add entities: Importer adds new entities or fails if entity exists.
+2. Add entities / update existing: Importer adds new entities or updaes existing entities.
+3. Update entities: Impoter updates existing entities or fails if entity does not exist.
+
+###### <a name="upload-screens-options-steps"></a>Steps
+
+1. Click on the next Button.
+
+![Upload file screen](images/plugin-upload/options-screen.png?raw=true, "Options")
+
+##### <a name="upload-screens-packages"></a>Packages page
+Because the entity (table) persons has no package defined, the by the packages page we have the posibility to choose another package then the MOLGENIS default package.
+The select options are generated from the available existing packages in this data set.
```

Comment:
"available existing" remove "existing", this is a bit overkill since a non-existing package would not be available.